client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx

```diff
@@ -55,10 +55,7 @@ const formatDate = (value: string): string | null => {
   return Formatter.naiveDate(DateUtils.getDateOrNull(value, 'dd/MM/yyyy'));
 };
 
-const formatNeedsReplacement = (value: string): boolean | null => {
-  if (value.match(/true/i)) return true;
-  else return false;
-};
+const formatNeedsReplacement = (value: string) => !!value.match(/true/i);
```

Comment:
I would suggest renaming the function to something a bit more explicit, like `parseStringToBoolean` or something, but up to you.

Reply:
This is what i call a stylistic change (up to developer preference), I think what you had originally works and look ok, but yeah agree that shorter version looks better. 

I think the important thing, that should not have been a nit: but change request, is the function name. I think the name should be 'isTruthyString'